docs/design/autoware-concepts/index.md

```diff
@@ -1,5 +1,40 @@
 # Autoware concepts
 
-!!! warning
+Autoware is the world’s first open-source software for autonomous driving systems. Autoware provides value for both The technology developers of autonomous driving systems can create new components based on Autoware. The service operators of autonomous driving systems, on the other hand, can select appropriate technology components with Autoware. This is enabled by the microautonomy architecture that modularizes its software stack into the core and universe subsystems (modules).
 
-    Under Construction
+## Microautonomy Architecture
+
+Autoware uses a [pipeline architecture](http://www.cs.sjsu.edu/~pearce/modules/patterns/distArch/pipeline.htm) to enable the development of autonomous driving systems. The pipeline architecture used in Autoware consists of components similar to [three-layer-architecture](http://www.flownet.com/gat/papers/tla.pdf). And they run in parallel. The components are designed to be extensible and reusable. And we call it microautonomy architecture.
+
+![core_and_universe.svg](core_and_universe.svg)
```

Comment:
TODO: Update icons after we decide them in https://github.com/autowarefoundation/.github/issues/2#issuecomment-1076443360.

Reply:
I've removed the icons till we decide them.